### ISSUE TITLE
master.json does not exist

### ISSUE THREAD

**Issue by lchuprasova** [scottsteinbeck/commandbox-gitbook#42]
Hi guys. I installed my development but I ran into an error while exporting.
![image](https://user-images.githubusercontent.com/76102776/103075525-5c734500-45fe-11eb-86da-4247492f0718.png)
there is really no such file in the archive that I downloaded. Tell me what to do?

**Comment by bdw429s** [comment]
@lchuprasova Thanks for the report.  It appears that Gitbook has modified the file names of their JSON files.  I have just pushed an updated version of the module (`1.1.13`).  Please run
```bash
update --system
```
in CommandBox, try again, and report back.  If you get any more errors, please report the full stack trace next time as well!

Merry Christmas!  :christmas_tree:

**Comment by lchuprasova** [comment]
You are awesome! PDF works now! But there are problems with encoding in PDF. Cyrillic is not written in PDF. Is there any way to solve this problem? Thank you, Merry Christmas!!!

**Comment by bdw429s** [comment]
@lchuprasova Unfortunatley, we've had issues in the past with other languages rendering properly.  You can see the comment thread on this ticket:

https://github.com/scottsteinbeck/commandbox-gitbook/issues/36

If you can upload the zip of your Gitbook export, I can take a look.  But I think the issues I hit last time were related to bugs in some of the 3rd party libraries we're using.

**Comment by lchuprasova** [comment]
@bdw429s  Thank you! Here is my gitbook archive:
https://drive.google.com/file/d/1_NpR-ZZOO90g-gAMupwS9QdinN-EccDS/view?usp=sharing

**Comment by lchuprasova** [comment]
Good day! Could not see the problem?

**Comment by bdw429s** [comment]
@lchuprasova Sorry for not replying!  Yes, I did download your gitbook archive a could weeks ago and I can see the same issue that's present in the other ticket I linked to.  Unfortunately, since this issue appears to exist in the underlying PDF library we're using I did enter a ticket with the underlying library
https://luceeserver.atlassian.net/browse/LDEV-3201
But I'll have to wait for them to issue a fix.
